flowkit_jwt_generator/flowkit_jwt_generator/jwt.py

```diff
@@ -172,23 +271,24 @@ def print_token(username, secret_key, lifetime):
 
     For example:
 
-    generate-jwt TEST_USER SECRET 1 --all-access http://localhost:9090
+    generate-jwt TEST_USER SECRET 1 TEST_SERVER --all-access http://localhost:9090
```

Comment:
Something for a future PR; just noticing that this command would be a little more readable if the first few arguments were passed using (mandatory) named arguments, too:
```
generate-jwt --all-access \
    --username TEST_USER \
    --secret-key SECRET \
    --lifetime 1 \
    --flowapi-server-name TEST_SERVER \
    http://localhost:9090
```